actionview/lib/action_view/railtie.rb

```diff
@@ -51,6 +51,14 @@ class Railtie < Rails::Engine # :nodoc:
       end
     end
 
+    initializer "action_view.setup_etag_with_template_digest" do |app|
+      ActiveSupport.on_load(:action_controller_base) do
+        etag do |options|
```

Comment:
This is only included in `ActionController::Base` so users can't exclude `EtagWithTemplateDigest` module. Only in their subclasses. About `ActionController::API`, yeah, we may need to document that, but I prefer to wait to see a report since I doubt people is using template digests in API controllers.